test/sequence-steps/github-release.js

```diff
@@ -0,0 +1,133 @@
+import test from "ava";
+import sinon from "sinon";
+import nodefn from "when/node";
+import { git } from "../helpers/index.js";
+
+const options = {
+	log: "* Added last feature\n* Added second feature\n* Added first feature",
+	versions: {
+		newVersion: "0.1.1"
+	},
+	username: "someUser",
+	token: "k0234f"
+
+};
+const utils = {
+	log: {
+		begin: sinon.spy(),
+		end: sinon.spy()
+	},
+	prompt: sinon.spy( prompterQuestions => new Promise( resolve => {
+		resolve( {
+			name: "My Special Release"
+		} );
+	} ) ),
```

Comment:
You should be able to use `sinon.stub().resolves( { name: "My Special Release" } )` because we have sinon-as-promised